.pre-commit-config.yaml

```diff
@@ -9,32 +9,36 @@ exclude: |
 
 repos:
   - repo: https://github.com/pre-commit/mirrors-clang-format
-    rev: v17.0.6
+    rev: v20.1.0
```

Comment:
Current Godot `master` is using `v19.1.3`. I'd prefer to not leap ahead of Godot. Or, is there a PR making this same update in Godot too?

Reply:
There was an update PR in waiting, but it wasn't merged when this was first opened